apps/web/playwright/teams.e2e.ts

```diff
@@ -12,30 +13,27 @@ test.describe("Teams", () => {
   test("Can create teams via Wizard", async ({ page, users }) => {
     const user = await users.create();
     const inviteeEmail = `${user.username}+invitee@example.com`;
+    let slug: string | RegExp;
     await user.apiLogin();
     await page.goto("/teams");
 
     await test.step("Can create team", async () => {
-      // Click text=Create Team
       await page.locator("text=Create Team").click();
       await page.waitForURL("/settings/teams/new");
-      // Fill input[name="name"]
       await page.locator('input[name="name"]').fill(`${user.username}'s Team`);
-      // Click text=Continue
+      slug = await page.locator(`[name='slug']`).innerText();
```

Comment:
is this needed?

Reply:
I added this check since there was no check. In the edge case scenario, it could happen that the URL changes after the save team. So I added this check, whatever the slug shows during creation should remain the same after saving.